internal/venice-common/src/main/java/com/linkedin/venice/pubsub/api/PubSubSecurityProtocol.java

```diff
@@ -0,0 +1,36 @@
+package com.linkedin.venice.pubsub.api;
+
+import java.util.Locale;
+
+
+/**
+ * This enum is equivalent to Kafka's SecurityProtocol enum.
+ *
+ * We need this abstraction because Kafka's enum is present in two different namespaces, which are different between
+ * LinkedIn's fork and the Apache fork.
+ */
+public enum PubSubSecurityProtocol {
+  /** Un-authenticated, non-encrypted channel */
+  PLAINTEXT(0, "PLAINTEXT"),
+  /** SSL channel */
+  SSL(1, "SSL"),
+  /** SASL authenticated, non-encrypted channel */
+  SASL_PLAINTEXT(2, "SASL_PLAINTEXT"),
+  /** SASL authenticated, SSL channel */
+  SASL_SSL(3, "SASL_SSL");
+
+  /** The permanent and immutable id of a security protocol -- this can't change, and must match kafka.cluster.SecurityProtocol  */
+  public final short id;
```

Comment:
Should we have this information in the Kafka-specific adapters?

Reply:
Possibly, yes. This code is copied from the Kafka enum. In our own code, we never reference this id, so I could delete it from our own enum...

Reply:
I removed this in the latest commit.